Palace/MyBooks/MyBooksDownloadCenter.swift

```diff
@@ -435,9 +435,10 @@ import OverdriveProcessor
 
 extension MyBooksDownloadCenter {
   func deleteLocalContent(for identifier: String, account: String? = nil) {
+    let current_account: String? = account ?? AccountsManager.shared.currentAccountId
```

Comment:
This is the main fix here, we were getting `nil` for `account` when deleting content, so we were unable to find the binaries.